client/src/Components/CommonComponent/Header/style.css

```diff
@@ -0,0 +1,16 @@
+.header i{
+  display: inline;
+  float: right;
+  margin-right: 2rem;
+  font-size: 2.5rem;
+}
+.header h2{
```

Comment:
use BEM the class name like that `header__h2`

Reply:
thanks shorouq :)